run_integration_test.go

```diff
@@ -33,7 +35,7 @@ func prepare(t *testing.T) func() {
 		os.RemoveAll(tmpdir)
```

Comment:
With a generated temp directory, this could be a simple `defer os.RemoveAll(tmpdir)`. No need for any conditionals.